src/shared/utils/flowSchemaNormalizer.ts

```diff
@@ -135,7 +135,9 @@ export const normalizeSchema = (
     data?.field
   )
 
-  const dedupedMeasurements = dedupeArray(measurements)
+  const dedupedMeasurements = dedupeArray(measurements).filter(a =>
```

Comment:
Oh man, I recently went through this code on my branch simplified it. Long story short, it would be more readable and understandable if we did something like this:

```
const setMeasurements = new Set(measurements)
const dedupedMeasurementsArray = [...setMeasurements]

```